containers/Dockerfile.buildenv

```diff
@@ -154,6 +154,23 @@ RUN wget https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.7.tar.gz -O bla
     && cd ../.. \
     && rm -rf yaml-cpp*
 
+# Install include-what-you-use
+RUN apt-get install -y libclang-10-dev
+WORKDIR /work
+RUN wget https://github.com/include-what-you-use/include-what-you-use/archive/clang_10.tar.gz \
+    && tar -xzf clang_10.tar.gz \
+    && rm clang_10.tar.gz \
+    && mkdir /work/include-what-you-use-clang_10/build \
+    && cd /work/include-what-you-use-clang_10/ \
+    && cd /work/include-what-you-use-clang_10/build \
+    && cmake -D CMAKE_CXX_COMPILER=clang++-10 \
+        -D CMAKE_C_COMPILER=clang-10 \
+        -D CMAKE_PREFIX_PATH=/usr/lib/llvm-10 .. \
```

Comment:
isn't `/usr/lib` in the default cmake prefix path, so you don't need to specify it?

Reply:
If I don't specify this, cmake picks up llvm-9 instead... presumably, though, if we had only installed one version of clang in the container then it would be as you say. I'll add a comment to clarify.